src/common/api/collectionsApi.ts

```diff
@@ -177,7 +177,7 @@ export type ColumnMeta = {
   display_name?: string;
 } & (
   | {
-      type: 'date' | 'int' | 'float';
+      type: 'date' | 'int' | 'float' | 'bool';
```

Comment:
unless this causes type issues, I would remove bool from this section. 

```suggestion
      type: 'date' | 'int' | 'float';
```